packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -54,7 +54,7 @@ export default decorate([
           version,
         } = this.props
         const { isTemplateInstalled } = this.state
-        if (getXoaPlan(process.env.XOA_PLAN) === 'Community') {
+        if (getXoaPlan(+process.env.XOA_PLAN) === 'Community') {
```

Comment:
This is the default value.

```suggestion
        if (getXoaPlan() === 'Community') {
```